src/test/java/ru/yandex/clickhouse/response/ClickHouseScrollableResultSetTest.java

```diff
@@ -0,0 +1,424 @@
+package ru.yandex.clickhouse.response;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import ru.yandex.clickhouse.settings.ClickHouseProperties;
+
+import java.io.ByteArrayInputStream;
+import java.sql.ResultSet;
+import java.util.TimeZone;
+
+import static org.testng.Assert.*;
+import static org.testng.AssertJUnit.assertEquals;
+
+public class ClickHouseScrollableResultSetTest {
+    @DataProvider(name = "longArrays")
+    public Object[][] longArrays() {
+        return new Object[][]{
+                {"[0]", new long[]{0}},
+                {"[324000111222,123,-456]", new long[]{324000111222L, 123, -456}},
+                {"[]", new long[]{}},
+        };
+    }
+
+    @Test(dataProvider = "longArrays")
+    public void toLongArrayTest(String str, long[] expected) throws Exception {
```

Comment:
I'm not sure the copied part must be exactly copied. Is it possible to do some parameterized test for common scenarios?

Reply:
Hi, I am not sure to understand you here. Do you want me to change the hardcoded response/values between the two test classes? Or do you want me to have shared tests between the two classes where the ResultSet instanciation should be done depending of the class were the test is called? 

Thanks.

Reply:
The latter one - to have some shared test for both ClickhouseResultSet and ClickHouseScrollableResultSet, and a special one for ClickHouseScrollableResultSet covering its specific features.

Reply:
The common tests are now at only one place.